scripts/minimal_init.lua

```diff
@@ -6,7 +6,6 @@ vim.o.swapfile = false
 vim.bo.swapfile = false
 
 require("nvim-treesitter.configs").setup {
-  ensure_installed = "maintained",
```

Comment:
was this needed? Maybe install "all"?

Reply:
It was not needed and just does random things when you launch the test. I think this was thought as a minimal rc for reports. But then it shouldn't be the same as the one for tests.